src/overlay_ordering.cpp

```diff
@@ -0,0 +1,25 @@
+#include "overlay_ordering.h"
+
+std::map<std::string, int> base_mutation_overlay_ordering;
+std::map<std::string, int> tileset_mutation_overlay_ordering;
+
+void load_overlay_ordering( JsonObject &jsobj )
```

Comment:
The base map is intended to provided a baseline for the order of all mutations, the data in mutation_ordering.json is used to populate the base map. The tileset map always overrides the setting in base, so tileset makers only need to update their specific tiles if the ordering is not the same. The base ordering is mostly an estimate of what order mutations may appear in.